schema/deploy/shipping/views.sql

```diff
@@ -1436,4 +1438,576 @@ grant select
+        household_4_tested,
+        household_4_result,
+        household_healthy_tested,
+        household_healthy_result
+
+      from warehouse.encounter
+      join warehouse.site using (site_id)
+      join warehouse.individual using (individual_id)
+      left join fu_illness using (encounter_id)
+      left join fu_change using (encounter_id)
+      left join fu_fever using (encounter_id)
+      left join fu_headache using (encounter_id)
+      left join fu_cough using (encounter_id)
+      left join fu_chills using (encounter_id)
+      left join fu_sweat using (encounter_id)
+      left join fu_throat using (encounter_id)
+      left join fu_nausea using (encounter_id)
+      left join fu_nose using (encounter_id)
+      left join fu_tired using (encounter_id)
+      left join fu_ache using (encounter_id)
+      left join fu_breathe using (encounter_id)
+      left join fu_diarrhea using (encounter_id)
+      left join fu_rash using (encounter_id)
+      left join fu_ear using (encounter_id)
+      left join fu_eye using (encounter_id)
+      left join fu_smell_taste using (encounter_id)
+      left join fu_feel_normal using (encounter_id)
+      left join fu_symptom_duration using (encounter_id)
+      left join fu_care using (encounter_id)
+      left join fu_date_care using (encounter_id)
+      left join fu_hospital_where using (encounter_id)
+      left join fu_hospital_ed using (encounter_id)
+      left join fu_work_school using (encounter_id)
+      left join fu_activities using (encounter_id)
+      left join fu_which_activities using (encounter_id)
+      left join fu_missed_activities using (encounter_id)
+      left join fu_test_result using (encounter_id)
+      left join fu_behaviors_no using (encounter_id)
+      left join fu_behaviors_inconclusive using (encounter_id)
+      left join fu_behaviors using (encounter_id)
+      left join fu_household_sick using (encounter_id)
+      left join fu_number_sick using (encounter_id)
+      left join fu_1_date using (encounter_id)
+      left join fu_1_symptoms using (encounter_id)
+      left join fu_1_test using (encounter_id)
+      left join fu_1_result using (encounter_id)
+      left join fu_2_date using (encounter_id)
+      left join fu_2_symptoms using (encounter_id)
+      left join fu_2_test using (encounter_id)
+      left join fu_2_result using (encounter_id)
+      left join fu_3_date using (encounter_id)
+      left join fu_3_symptoms using (encounter_id)
+      left join fu_3_test using (encounter_id)
+      left join fu_3_result using (encounter_id)
+      left join fu_4_date using (encounter_id)
+      left join fu_4_symptoms using (encounter_id)
+      left join fu_4_test using (encounter_id)
+      left join fu_4_result using (encounter_id)
+      left join fu_healthy_test using (encounter_id)
+      left join fu_healthy_result using (encounter_id)
+      left join warehouse.encounter initial on initial.identifier = encounter.details ->> 'part_of'
```

Comment:
I believe `encounter.details @> jsonb_build_object('part_of', initial.identifier)` would use the index on `details`, but only a query plan would tell for sure.

Reply:
In this case `initial.identifier = encounter.details ->> 'part_of'` is much faster than `encounter.details @> jsonb_build_object('part_of', initial.identifier)`. I uploaded the query plans [here](https://gist.github.com/joverlee521/bba66984f74f6970cf11439bd2afd51c) if you're curious.

Reply:
Thanks!